packages/content/src/components/content-header/types.ts

```diff
@@ -0,0 +1,12 @@
+import { ContentKind } from '@thoughtindustries/content/src/graphql/global-types';
+
+export interface ContentHeaderProps {
```

Comment:
Any particular reason to use an `interface` vs `type` here? Also a head's up, since these are the props for the `ContentHeader` component, think it'd be fine to import the type for use here:

https://github.com/thoughtindustries/helium/blob/7ee45955c4672ae73a8d47ba894e7bf546cec999/packages/content/src/components/content-header/content-header.tsx#L43